cylc.language-configuration.json

```diff
@@ -27,5 +27,92 @@
         ["\"", "\""],
         ["'", "'"],
         ["<", ">"]
-    ]
+    ],
+	"scheduling section": {
+		"prefix": ["scheduling", "schd"],
+		"body": [
+			"[scheduling]",
+			"    initial cycle point = ${1:1066}",
+			"    [[graph]]",
+			"        ${2:R1} = ${3:foo}"
+		]
+	},
+	"task": {
+		"prefix": ["task"],
+		"body": [
+			"    [[${1:root}]]",
+			"        script = \"\"\"",
+			"            ${2:true}",
+			"        \"\"\"",
+			"        platform = ${3:}",
+			"        execution time limit = ${4:PT1M}",
+			"        execution retry delays = ${5:PT1M}",
+		],
+		"description": ["Pre formed task section"]
+	},
+	"spice": {
+		"prefix": ["spice"],
+		"body": [
+			"    [[${1:root}]]",
+			"        script = \"\"\"",
+			"            ${2:true}",
+			"        \"\"\"",
+			"        platform = ${3:spice}",
+			"        execution time limit = ${4:PT1M}",
+			"        execution retry delays = ${5:PT1M}",
+			"        [[[directives]]]",
+			"            --mem=${6:256M}",
+			"            --ntasks=${7:2}",
+		],
+		"description": ["Pre formed task section for a spice task"]
+	},
```

Comment:
Met Office specific stuff shouldn't be in the extension. Instead we could document internally how users can add this snippet themselves.

Reply:
This whole section was a mistake - I meant only to commit the auto generated thing...